pkg/importer/vddk-datasource.go

```diff
@@ -378,12 +439,32 @@ func (vs *VDDKDataSource) TransferFile(fileName string) (ProcessingPhase, error)
 		}
 
 		// Only log progress at approximately 1% intervals.
-		currentProgress := uint(100.0 * (float32(i+uint64(written)) / float32(size)))
-		if currentProgress > lastProgress {
-			klog.Infof("Transferred %d/%d bytes (%d%%)", i+uint64(written), size, currentProgress)
-			lastProgress = currentProgress
+		currentProgressBytes := i + uint64(written)
+		currentProgressPercent := uint(100.0 * (float32(currentProgressBytes) / float32(size)))
```

Comment:
Any particular reason you didn't use float64 here? Not entirely sure what the max value of a float32 is, but it might not be big enough for large VMs?

Reply:
No reason, and looking around I see people usually use float64 for this exact purpose. It's not a big deal to change it, so I will do that.